tests/grpc/client/tx/sign.go

```diff
@@ -24,7 +24,8 @@ func BroadcastTx(clientCtx client.Context, txf tx.Factory, msgs ...sdk.Msg) (*sd
 		return nil, err
 	}
 
-	txf = txf.WithGas(adjusted)
+	// make sure gas in enough to execute the txs
+	txf = txf.WithGas(adjusted * 2)
```

Comment:
can you check if `1.5` will be enough? If you need to do integer than do: `adjusted * 3 / 2`